sdk/core/core-http/src/webResource.ts

```diff
@@ -637,8 +699,17 @@ export interface RequestPrepareOptions {
    * Indicates whether the request body is a stream (useful for file upload scenarios).
    */
   bodyIsStream?: boolean;
+  /**
+   * Signal of an abort controller. Can be used to abort both sending a network request and waiting for a response.
```

Comment:
Can you copy this comment over the property on the class itself? It is slightly more detailed the one above.